projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-column-modal/documenten-api-column-modal.component.html

```diff
@@ -0,0 +1,135 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<cds-modal
+  valtimoCdsModal
+  *ngIf="{
+    isAdd: isAdd$ | async,
+    isEdit: isEdit$ | async,
+    isClosed: isClosed$ | async,
+    editingKey: editingKey$ | async,
+    disabled: disabled$ | async
+  } as obs"
+  [open]="!obs.isClosed"
+  size="sm"
+>
+  <cds-modal-header showCloseButton="true" (closeSelect)="onClose()">
+    <h3 cdsModalHeaderHeading>
+      {{ obs.isAdd ? ('dossierManagement.statuses.add' | translate) : '' }}
+      {{ obs.isEdit ? ('dossierManagement.statuses.edit' | translate) : '' }}
+    </h3>
+  </cds-modal-header>
+
+  <section cdsModalContent>
+    <ng-container *ngTemplateOutlet="statusForm; context: {obs: obs}"></ng-container>
+  </section>
+
+  <cds-modal-footer>
+    <ng-container *ngTemplateOutlet="buttons; context: {obs: obs}"></ng-container>
+  </cds-modal-footer>
+</cds-modal>
+
+<ng-template #colorDropdownItemTemplate let-item="item">
+  <div class="dropdown-color-tag">
+    <cds-tag *ngIf="item && item.content" class="cds-tag--no-margin" [type]="item?.tagType">
+      {{ 'interface.tagType.' + item?.tagType | translate }}
+    </cds-tag>
```

Comment:
Already checking for the truthy value of item so no need for the "?"

```suggestion
    <cds-tag *ngIf="!!item && item.content" class="cds-tag--no-margin" [type]="item.tagType">
      {{ 'interface.tagType.' + item.tagType | translate }}
    </cds-tag>
```